client/js/components/procedure/basicSettings/DpAllowedSenderEmailList.vue

```diff
@@ -8,13 +8,17 @@
 </license>
 
 <template>
-  <dp-email-list
-    :init-emails="allowedEmailAddresses"
-    form-field-name="allowedSenderEmailAddresses[][fullAddress]" />
+  <div>
```

Comment:
This whole component, along with the twig code in the block starting at projects/ewm/templates/bundles/DemosPlanProcedureBundle/DemosPlanProcedure/administration_edit.html.twig:124, should be also moved into the addon. however i guess practical reasons speak against that?